test/instances/morpho-blue/MorphoBlue_cbETH_USDC_Base_Hyperdrive.t.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { stdStorage, StdStorage } from "forge-std/Test.sol";
+import { IMorpho } from "morpho-blue/src/interfaces/IMorpho.sol";
+import { IERC20 } from "../../../contracts/src/interfaces/IERC20.sol";
+import { IFiatTokenProxy } from "../../../contracts/src/interfaces/IFiatTokenProxy.sol";
+import { IHyperdrive } from "../../../contracts/src/interfaces/IHyperdrive.sol";
+import { IMorphoBlueHyperdrive } from "../../../contracts/src/interfaces/IMorphoBlueHyperdrive.sol";
+import { Lib } from "../../utils/Lib.sol";
+import { MorphoBlueHyperdriveInstanceTest } from "./MorphoBlueHyperdriveInstanceTest.t.sol";
+
+contract MorphoBlue_cbETH_USDC_Base_HyperdriveTest is
+    MorphoBlueHyperdriveInstanceTest
+{
+    using Lib for *;
+    using stdStorage for StdStorage;
+
+    /// @dev The address of the loan token. This is just the USDC token.
+    address internal constant LOAN_TOKEN =
+        address(0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913);
+
+    /// @dev Whale accounts.
+    address internal LOAN_TOKEN_WHALE =
+        address(0x3304E22DDaa22bCdC5fCa2269b418046aE7b566A);
+    address[] internal baseTokenWhaleAccounts = [LOAN_TOKEN_WHALE];
+
+    /// @notice Instantiates the instance testing suite with the configuration.
+    constructor()
+        MorphoBlueHyperdriveInstanceTest(
+            InstanceTestConfig({
+                name: "Hyperdrive",
+                kind: "MorphoBlueHyperdrive",
+                decimals: 6,
+                baseTokenWhaleAccounts: baseTokenWhaleAccounts,
+                vaultSharesTokenWhaleAccounts: new address[](0),
+                baseToken: IERC20(LOAN_TOKEN),
+                vaultSharesToken: IERC20(address(0)),
+                // NOTE: The share tolerance is quite high for this integration
+                // because the vault share price is ~1e12, which means that just
+                // multiplying or dividing by the vault is an imprecise way of
+                // converting between base and vault shares. We included more
+                // assertions than normal to the round trip tests to verify that
+                // the calculations satisfy our expectations of accuracy.
+                shareTolerance: 1e3,
+                minimumShareReserves: 1e6,
+                minimumTransactionAmount: 1e6,
+                positionDuration: POSITION_DURATION,
+                fees: IHyperdrive.Fees({
+                    curve: 0.001e18,
+                    flat: 0.0001e18,
```

Comment:
These fees look off to me. don't we usually do 1% curve and .05% flat?

Reply:
oh sorry, this is just the test

Reply:
No worries, yeah I was shooting for low fees for this test to make the test more conservative.